examples/Learning_example/learning_example.ipynb

```diff
@@ -0,0 +1,2295 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    X1 = np.asarray(node1.model.train_dataloader.dataset.X)
</pre><p>Why is t here no training data at X=0 neither for node1 or node2? Won't that mess up test accuracy near X=0?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>We can add so we dont have gaps.</p>